postman_swagger.md

```diff
@@ -0,0 +1,70 @@
+
+**11. Click Import and wait**
+
+![](https://ajeuwbhvhr.cloudimg.io/colony-recorder.s3.amazonaws.com/files/2023-07-11/9c6f9e99-8fab-4198-9de7-208ceb950e4d/ascreenshot.jpeg?tl_px=585,617&br_px=2078,1457&force_format=png&width=560&wat_scale=50&wat=1&wat_opacity=0.7&wat_gravity=northwest&wat_url=https://colony-recorder.s3.us-west-1.amazonaws.com/images/watermarks/FB923C_standard.png&wat_pad=262,139)
+
+**12. When Import is Done, you will be able to see MassaStation APi imported, and you can browse all it's endpoints**
+
+![](https://ajeuwbhvhr.cloudimg.io/colony-recorder.s3.amazonaws.com/files/2023-07-11/d76fba29-49e4-41c0-bbaa-2728a6cd79c9/ascreenshot.jpeg?tl_px=0,16&br_px=1493,856&force_format=png&width=560&wat_scale=50&wat=1&wat_opacity=0.7&wat_gravity=northwest&wat_url=https://colony-recorder.s3.us-west-1.amazonaws.com/images/watermarks/FB923C_standard.png&wat_pad=59,139)
+
+**13. let's try to call an endpoint, but before you should have your massa station running in background, you can run it directly from the terminal, inside station repo, type 
+`go run cmd/massastation/main.go `**
+
+![](https://ajeuwbhvhr.cloudimg.io/colony-recorder.s3.amazonaws.com/files/2023-07-11/fc46686c-7f76-4e55-ae67-f86ea77fe124/ascreenshot.jpeg?tl_px=1117,1124&br_px=2610,1964&force_format=png&width=560&wat_scale=50&wat=1&wat_opacity=0.7&wat_gravity=northwest&wat_url=https://colony-recorder.s3.us-west-1.amazonaws.com/images/watermarks/FB923C_standard.png&wat_pad=262,288)
+
+**14. Expand MassaStation folder, and let's try to call an endpoint**
+
+![](https://ajeuwbhvhr.cloudimg.io/colony-recorder.s3.amazonaws.com/files/2023-07-11/e558a066-72bf-4e88-8775-bdfcf583bc9c/ascreenshot.jpeg?tl_px=0,299&br_px=1493,1139&force_format=png&width=560&wat_scale=50&wat=1&wat_opacity=0.7&wat_gravity=northwest&wat_url=https://colony-recorder.s3.us-west-1.amazonaws.com/images/watermarks/FB923C_standard.png&wat_pad=175,139)
+
+**15. replace the `{{baseUrl}}` by `station.massa`**
+
+![](https://ajeuwbhvhr.cloudimg.io/colony-recorder.s3.amazonaws.com/files/2023-07-11/7afc05d9-5cf5-4985-86f2-683a314b879e/ascreenshot.jpeg?tl_px=399,48&br_px=1892,888&force_format=png&width=560&wat_scale=50&wat=1&wat_opacity=0.7&wat_gravity=northwest&wat_url=https://colony-recorder.s3.us-west-1.amazonaws.com/images/watermarks/FB923C_standard.png&wat_pad=262,139)
+
+**16. click on send, and wait few seconds ...**
+
+![](https://ajeuwbhvhr.cloudimg.io/colony-recorder.s3.amazonaws.com/files/2023-07-11/51e5c1f9-b4fd-4a60-b90a-fd72c29a57d5/ascreenshot.jpeg?tl_px=1530,68&br_px=3023,908&force_format=png&width=560&wat_scale=50&wat=1&wat_opacity=0.7&wat_gravity=northwest&wat_url=https://colony-recorder.s3.us-west-1.amazonaws.com/images/watermarks/FB923C_standard.png&wat_pad=446,139)
+
+**17. You will receive in response all websites uploaded so far to the current DNS.**
```

Comment:
this endpoint take a while to respond maybe it's not the best one to demonstrate Massa Station. @SlnPons WDTY?

Reply:
we are working on it to fix the delay , but it's the more representative endpoint to get some significant data ..